tasks/deploy/lib/constants.ts

```diff
@@ -23,3 +26,5 @@ export const MAINNET_STETH_WHALE =
 
 export const MAINNET_RETH_WHALE =
     "0x1bee69b7dfffa4e2d53c2a2df135c388ad25dcd2" as Address;
+
+export const SIX_MONTHS = "182 days";
```

Comment:
why not 525,600/2 minutes?

Reply:
![ed3e1892-63bf-4a63-8028-2a5cc4543613_text](https://github.com/delvtech/hyperdrive/assets/5555162/f6d5290a-1fec-475d-a5ce-f692df99c9e5)

Reply:
It's not divisible by a day